erpnext/accounts/doctype/purchase_invoice/purchase_invoice.json

```diff
@@ -3384,7 +3384,7 @@
    "bold": 0, 
    "collapsible": 0, 
    "columns": 0, 
-   "fieldname": "subscription", 
+   "fieldname": "subscription_id", 
```

Comment:
noooo! Leave this as `subscription` link field naming is very important!

Reply:
Fixed!